services/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -37,7 +37,7 @@ class ProjectService(projectRepository: ProjectRepository, projectVersioning: Pr
       )
     projectVersioning.createRepository(localProject).flatMap {
       case Left(exception) => Future.successful(Left(exception))
-      case Right(project)  => projectRepository.addProject(project).map(Right(_))
+      case Right(project)  => projectRepository.addProject(project).map(prId => Right(project.copy(projectId = prId)))
```

Comment:
```suggestion
      case Right(project)  => projectRepository.addProject(project).map(_ => Right(project))
```
This call of `copy` seems unnecessary. I can't image any case when project repository will suddenly decide to replace `projectId` with something else.
Alternatively you can return whole project from `repository` too, then it will look even simpler
```suggestion
      case Right(project)  => projectRepository.addProject(project).map(Right(_))
```